src/main/java/net/openhft/chronicle/core/util/StringUtils.java

```diff
@@ -377,7 +381,8 @@ private static boolean compareRest(@NotNull CharSequence in,
         return true;
     }
 
-    private static double asDouble(long value, int exp, boolean negative, int decimalPlaces) {
+    @Deprecated(/* For removal in x.25 */)
+    public static double asDouble(long value, int exp, boolean negative, int decimalPlaces) {
```

Comment:
Just expose this one for the purpose of the comparison

Reply:
can we remove in x.24?

Reply:
i.e. as part of this PR?